### PR TITLE
Added conda install install instructions.

### PR DESCRIPTION
**Pull request by ocefpaf**
@dblodgett-usgs here are a few lines with the conda instructions to install pyGDP using the [IOOS](https://binstar.org/ioos) channel.  I do not wish to pollute the README with conda installation instructions, so I just added a link to the [IOOS wiki](https://github.com/ioos/conda-recipes/wiki).  There people can find detailed install instructions for Linux, MacOS, and Windows.

PS: The requirement files is in Markdown format.  I think that the original intention was to convert the README (See https://github.com/USGS-CIDA/pyGDP/issues/72) to Markdown and not the requirement file (which is rendered unusable as Markdown).

I Can fix that for you in this PR if you wish.

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -62,6 +62,17 @@ Find and install the lxml and owslib though Package Manager
 (Recommended you set Canopy as your default python environment)
 Install pyGDP as in 1.)
 
+OR
+4.) Install using Anaconda/miniconda
+Add the IOOS channel and install pyGDP:
+    conda config --add channels ioos -f
+    conda install pygdp
+
+Install pyGDP without adding the IOOS channel:
```
Comment (jread-usgs) [inline README.txt]: probably just use the install w/o subscribe?

**Comment by dblodgett-usgs** [comment]
Yes, please do fix my carelessness with the requirements.txt to .md! Woops!

**Comment by ocefpaf** [comment]
@jread-usgs and @dblodgett-usgs is this OK?  I can squash the commits if you want.

**Comment by dblodgett-usgs** [comment]
Looks good to me. Jordan, sounds like you know conda better than I. Will let you merge.

**Comment by jread-usgs** [comment]
whoops, thanks for merging @dblodgett-usgs

**Pull request merged**